Core/PoTItem.cs

```diff
@@ -433,9 +439,10 @@ public void Roll(int itemLevel)
 	public static void GenerateItemList()
 	{
 		AllItems.Clear();
+
 		foreach (Type type in PathOfTerraria.Instance.Code.GetTypes())
 		{
-			if (type.IsAbstract || !type.IsSubclassOf(typeof(PoTItem)))
+			if (type.IsAbstract || !type.IsSubclassOf(typeof(PoTItem)) || Attribute.IsDefined(type, typeof(ManuallyLoadPoTItemAttribute)))
```

Comment:
Had another thought - Is this to prevent them from showing up in items that get dropped in our system? So they can only be obtained by the normal vanilla means?

This is a good thing ; Just asking

Reply:
This attribute stops manually loaded items - at the moment, just InstancedVanillaClone - from being autoloaded, as InstancedVanillaClone has a ctor which takes any parameters, whereas the Activator.CreateInstance used for autoload expects a parameterless ctor. This would also be nice for any other manual loading, we could switch something like Battleaxes for example to a template class and manually load them instead of making individual classes since, outside of uniques, Battleaxes do not do anything special and it'd remove boilerplate.